docs/backend_readme.md

```diff
@@ -235,4 +235,18 @@ To track the historical events for a certain model based on their id and timesta
 Context.objects.filter(metadata__tracked_model_id=tracked_model_id).order_by("metadata__timestamp")
 ```
 
-</details>
+---
+
+### Troubleshooting
+
+#### Aliases
+
+For a list of command aliases, refer to [.bash_aliases](https://github.com/BetterAngelsLA/monorepo/blob/main/.bash_aliases)
+
+#### Resetting DB
+
+If an error occurs where the local database doesn't accurately reflect the new migration(s), run:
```

Comment:
Since this is a destructive action, perhaps we could add a note that this will `drop the database` and then recreate it (with seed data), so all data will be reset.